ext/css/display.css

```diff
@@ -288,6 +288,12 @@ a {
     color: var(--link-color);
     text-decoration: underline;
     cursor: pointer;
+    text-underline-offset: 0.2rem;
+    text-decoration-thickness: 2px;
```

Comment:
Use em units instead of px. Feel free to use `calc` and `--font-size-no-units`
```css
calc(2em / var(--font-size-no-units))
```

Reply:
@toasted-nutbread Is that really necessary for an underline? I believe most uses use their browser's zoom functionality (which increases the size of `px`) if they want non-text UI elements to increase in size. I feel like a user just increasing font size (i.e., increasing the size of `rem` and `em`) without zooming should not increase the width of lines.

Reply:
Underline size correlates with the font size, so it should scale with it in general. If you look carefully at the CSS of this extension, you'll notice that there is like 1 place where `px` units are used. The reason for this is to allow some of the scaling functionality to look correct when using mobile devices and the visual viewport. The layout is scaled by adjusting the root font size. In general, this was easier than using a transform to accomplish the same thing, since transform doesn't actually affect the layout, but the scaling popup window needs to be affected by its size. There are also sometimes some negative effects of using scaling where the text will not be antialiased nicely.

Reply:
Example:
font-size at 100px, underline thickness at 2px:
![image](https://github.com/themoeway/yomitan/assets/11037431/384c9a11-94d1-4c74-b549-bd9741c40b8c)

underline thickness not specified:
![image](https://github.com/themoeway/yomitan/assets/11037431/9971b93b-1251-4414-9ea5-0a0bfc3a774c)

Reply:
I had for some reason confused myself into thinking that built-in text-decoration underline doesn't scale with font-size by default, but it does. In that case your suggestion makes perfect sense, simply to align with default browser behavior if nothing else -- thanks!